src/test/kotlin/ControllerTest.kt

```diff
@@ -18,68 +21,74 @@ import javax.servlet.http.HttpServletRequest
 class ControllerTest {
     @Test
     fun test_generation_of_urls_for_new_begrep() {
-        val sqlStoreMock: SqlStore = prepareSqlStoreMock()
+        val begrepRepositoryMock: BegrepRepository = prepareBegrepRepositoryMock()
         val httpServletRequestMock: HttpServletRequest = mock()
+        val postRequestUri = "/begreper"
+        whenever(httpServletRequestMock.requestURI).thenReturn(postRequestUri)
+
         val fdkPermissionsMock: FdkPermissions = prepareFdkPermissionsMock()
 
-        val begreperApiImplK = BegreperApiImplK(sqlStoreMock, fdkPermissionsMock)
+        val begreperApiImplK = BegreperApiImplK(begrepRepositoryMock, fdkPermissionsMock)
         begreperApiImplK.baseURL = "https://registrering-begrep.ut1.fellesdatakatalog.brreg.no"
         val begrep = Begrep()
-        begrep.ansvarligVirksomhet= createTestVirksomhet()
+        begrep.ansvarligVirksomhet = createTestVirksomhet()
 
         val retValue = begreperApiImplK.createBegrep(httpServletRequestMock, begrep)
         assertNotNull(retValue)
         assertNotNull(retValue.headers)
-        assertEquals( begreperApiImplK.baseURL + makeBegrep().ansvarligVirksomhet.id + "/" + makeBegrep().id, retValue.headers.get("Location")?.get(0))
-
+        val locationHeaderValue: String? = retValue.headers.get("Location")?.get(0);
+        val testRegExp = Regex("^/begreper/[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$", RegexOption.IGNORE_CASE);
```

Comment:
Forslag/ynskje: skrive ein kommentar som dokumenterer kva denne regexp skal gjere (for oss som ikkje er fluent i regexp), ver så snill.

Reply:
Ok

Reply:
Done